target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-whw01-v1.dts

```diff
@@ -0,0 +1,237 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "qcom-ipq4019-ap.dk01.1.dtsi"
+#include <dt-bindings/leds/common.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	model = "Linksys WHW01 v1";
+	compatible = "linksys,whw01-v1";
+
+	aliases {
+		led-boot = &led_system_blue;
+		led-running = &led_system_blue;
+	};
+
+	chosen {
+		bootargs-append = " root=/dev/ubiblock0_0";
+	};
+
+	reserved-memory {
+		rsvd1@87000000 {
+			reg = <0x87000000 0x500000>;
+			no-map;
+		};
+
+		wifi_dump@87500000 {
+			reg = <0x87500000 0x600000>;
+			no-map;
+		};
+
+		rsvd2@87B00000 {
+			reg = <0x87b00000 0x300000>;
+			no-map;
+		};
+	};
+
+	soc {
+		gpio_keys {
+			pinctrl-0 = <0x52>;
+			pinctrl-names = "default";
+			compatible = "gpio-keys";
+
+			button@1 {
```

Comment:
"reset"?

Reply:
Renamed to `reset` to match other devices.